src/Faker/Calculator/Isbn.php

```diff
@@ -34,7 +34,7 @@ public static function checksum(string $input): string
         $digits = str_split($input);
         array_walk(
             $digits,
-            static function (&$digit, $position) {
+            static function (& $digit, $position) {
```

Comment:
How is this relevant to the PR? Please revert

Reply:
This was due to the CS script.

Reply:
Which one?